Common/Securities/CashBuyingPowerModel.cs

```diff
@@ -89,7 +95,8 @@ public override HasSufficientBuyingPowerForOrderResult HasSufficientBuyingPowerF
             if (parameters.Order.Direction == OrderDirection.Sell)
             {
                 // can sell available and non-reserved quantities
-                isSufficient = orderQuantity <= totalQuantity - openOrdersReservedQuantity;
+                var availableQuantity = Math.Round(totalQuantity - openOrdersReservedQuantity, amountPrecision);
```

Comment:
I think `totalQuantity - openOrdersReservedQuantity` shouldn't require rounding, since the current holding and the open orders should already respect the lot size, else it means there is some other bug

Reply:
I agree, it's a fix of a symptom vs actual issue.

I think total quantity can have invalid value because of Portfolio manipulations (ProcessFill method) that don't respect lot size i.e they don't round amounts after multiplications.

But here it's a '-' so shouldn't require Math.Round